lib/scavenger_hunt/app/assets/stylesheets/scavenger_hunt/components/_typography.sass

```diff
@@ -9,6 +9,7 @@ h2, .subtitle
   font-weight: 600
   margin: 0 0 $superview 0
   text-transform: uppercase
+  padding-top: 10px
```

Comment:
Is there a reason you didn't use `$gutter` here for visual consistency?

Reply:
@flipsasser Nope! No reason other than just not thinking about variables. I'm noting to always look for a variable first after all this awesome fedback. Thanks so much! Updated!